fullcalendarcomponent/fullcalendar/fullcalendar.js

```diff
@@ -336,6 +354,18 @@ angular.module('svyFullcalendar', ['servoy']).directive('svyFullcalendar', funct
 					if ($scope.handlers.onEventRightClickMethodID) {
 						element.bind('contextmenu', { event: event }, eventRightClick);
 					}
+
+					// showBgEventsTitle is a custom option defined when calendar is initialized
+					// fc-bg-title is a custom css class just to differentiate from fc-title
+					// FIXME is this the best way to do it? 
+					if (event.rendering == 'background' && event.title && $scope.model.calendarOptions.showBgEventTitle) {
+						element.append('<div class="fc-bg-title"><span>' + event.title + '</span></div>');
```

Comment:
What happens when the text is too long ?
Do we need some styling to prevent overflow ?

Reply:
The background event will be render as the regular events, the text is wrapped.
I don't think it is necessary to implement any styling